src/locale/locales/it/messages.po

```diff
@@ -133,89 +121,71 @@ msgstr "{0} di {1}"
 
 #: src/screens/StarterPack/StarterPackScreen.tsx:479
 msgid "{0} people have used this starter pack!"
-msgstr "{0} persone hanno usat questo starter pack!"
+msgstr "{0} persone hanno usato questo starter pack!"
 
 #: src/view/shell/bottom-bar/BottomBar.tsx:203
 msgid "{0} unread items"
-msgstr ""
-
-#~ msgid "{0} your feeds"
-#~ msgstr "{0} tuoi feed"
+msgstr "{0} elementi non letti"
 
 #: src/view/com/util/UserAvatar.tsx:435
 msgid "{0}'s avatar"
 msgstr "Foto profilo di {0}"
 
-#: src/screens/StarterPack/Wizard/StepDetails.tsx:68
+#: src/screens/StarterPack/Wizard/StepDetails.tsx:67
 msgid "{0}'s favorite feeds and people - join me!"
 msgstr "Feed e utenti preferiti di {0} - unisciti!"
 
-#: src/screens/StarterPack/Wizard/StepDetails.tsx:47
+#: src/screens/StarterPack/Wizard/StepDetails.tsx:46
 msgid "{0}'s starter pack"
 msgstr "Starter pack di {0}"
 
 #. How many days have passed, displayed in a narrow form
 #: src/lib/hooks/useTimeAgo.ts:158
 msgid "{0}d"
-msgstr ""
+msgstr "{0}g"
 
 #. How many hours have passed, displayed in a narrow form
 #: src/lib/hooks/useTimeAgo.ts:148
 msgid "{0}h"
-msgstr ""
+msgstr "{0}o"
```

Comment:
Here I would leave "{0}h", which seems to me to be consolidated even in Italian. It is preserved even with the javascript libraries.
![image](https://github.com/user-attachments/assets/527a2ea4-70e7-4c67-8667-b5bb4c37a8a3)
What do you think?

Reply:
I agree. Furthermore [also CLDR uses `h`](https://github.com/unicode-org/cldr/blob/main/common/main/it.xml#L2923) (CLDR is a widely used Unicode library - for example by Android and iPhone devices) - see 5b99f9b5e45f290e2092e3b2ca7697d92d56c0a3